### DISCUSSION TITLE
Support for FS100 controller

### DISCUSSION THREAD

**Discussion by ashwinsnambiar** [Yaskawa-Global/motoros2#326] [Q&A] [answered]
Hello!

Is the ROS2 support for FS100 controllers coming down the pipeline in the future? If yes, around when can it be expected?

Thank you.

**Comment by gavanderhoorn** [comment, marked as answer]
Please see #19 and the recently closed #227 for the current status.

**Reply by ashwinsnambiar** [reply]
Thanks for the quick response. I searched the "discussions" alone for the same question. Should have checked the "issues" too. Sorry for that.

**Reply by gavanderhoorn** [reply]
No need to apologise.

Could you tell us a little bit about your use-case for MotoROS2 with an FS100? Knowing more about what people would potentially want to do with such controllers could help us prioritise development.

**Reply by ashwinsnambiar** [reply]
Yeah sure. I am currently working in a lab in Germany. We have 3 robots using the FS100 controllers: SIA10F, SIA20F and a GP series robot. Currently, we are running the robot cell using the ROS1 drivers but we are planning to migrate to ROS2 for future developments.
